library/src/main/java/net/rdrei/android/dirchooser/DirectoryChooserFragment.java

```diff
@@ -330,20 +369,23 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     /**
      * Shows a confirmation dialog that asks the user if he wants to create a
-     * new folder.
+     * new folder. User can modify provided name, if it was not disallowed.
      */
     private void openNewFolderDialog() {
-        new AlertDialog.Builder(getActivity())
+        View dialogView = getActivity().getLayoutInflater().inflate(R.layout.dialog_new_folder, null);
```

Comment:
Yup, totally right. I merged it with a suppress annotation. Thanks again
for your work! :)

On Mon, Jul 27, 2015, 17:29 Majeur <notifications@github.com> wrote:

> In
> library/src/main/java/net/rdrei/android/dirchooser/DirectoryChooserFragment.java
> <https://github.com/passy/Android-DirectoryChooser/pull/55#discussion_r35556103>
> :
>
> >       */
> >      private void openNewFolderDialog() {
> > -        new AlertDialog.Builder(getActivity())
> > +        View dialogView = getActivity().getLayoutInflater().inflate(R.layout.dialog_new_folder, null);
>
> Parent argument is used by the inflater to determine and use appropriate
> LayoutParams when inflating. Then LayoutInflater is retrieved from the
> activity so it inherit the app's style.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/passy/Android-DirectoryChooser/pull/55/files#r35556103>
> .
>